pfcpiface/node.go

```diff
@@ -55,6 +55,21 @@ func (node *PFCPNode) handleNewPeers() {
 	lAddrStr := node.LocalAddr().String()
 	log.Infoln("listening for new PFCP connections on", lAddrStr)
 
+	for _, peer := range node.upf.peers {
+		log.Println("SPGWC/SMF hostname ", peer)
```

Comment:
Two remarks:
- please use `log.Info` instead of `log.Println`
- please use log contexts (`log.WithFields()`)

Reply:
This change is done and also fixed golangci linter errors.